{
  "unversionedId": "error_handling",
  "id": "error_handling",
  "title": "Error Handling",
  "description": "oclif handles intentionally - and unintentionally - thrown errors in two places. First in the Command.catch method and then, finally, in the bin/run catch handler where the Error is printed and the CLI exits. This error flow makes it possible for you to control and respond to errors that occur in your CLI as you see fit.",
  "source": "@site/../docs/error_handling.md",
  "sourceDirName": ".",
  "slug": "/error_handling",
  "permalink": "/docs/error_handling",
  "draft": false,
  "editUrl": "https://github.com/oclif/oclif.github.io/tree/docs/docs/../docs/error_handling.md",
  "tags": [],
  "version": "current",
  "lastUpdatedBy": "Author",
  "lastUpdatedAt": 1539502055,
  "formattedLastUpdatedAt": "Oct 14, 2018",
  "frontMatter": {
    "title": "Error Handling"
  },
  "sidebar": "docs",
  "previous": {
    "title": "Help Classes",
    "permalink": "/docs/help_classes"
  },
  "next": {
    "title": "JSON",
    "permalink": "/docs/json"
  }
}